lib/common.sh

```diff
@@ -11,7 +11,20 @@ DAPP_LIB=${DAPP_LIB:-$BIN_DIR/contracts}
 
 export OUT_DIR=${OUT_DIR:-$PWD/out}
 
-CONFIG_FILE="$CONFIG_DIR/$CONFIG_STEP.json"
+CONFIG_FILE="${OUT_DIR}/config.json"
+# Only executes if called from the initial script
+if [[ "$CONFIG_STEP" = "$1" ]]; then
+    # Clean out directory
+    rm -rf "$OUT_DIR"
```

Comment:
Maybe this could be `rm -rf "${OUT_DIR}/*"`?

This way you wouldn't need the `mkdir` right after the deletion.

Reply:
First time you will need to create it though, or if for any case it was removed.
So I think we will still need the `mkdir` line (with some conditional).